src/ontodia/workspace/resizableSidebar.tsx

```diff
@@ -35,20 +35,25 @@ export class ResizableSidebar extends React.Component<Props, State> {
         minWidth: 0,
         maxWidth: 500,
         defaultWidth: 275,
-        initiallyOpen: true,
+        isOpen: true,
     };
 
     private originWidth: number;
 
     constructor(props: Props) {
         super(props);
-        const {initiallyOpen} = this.props;
         this.state = {
-            open: initiallyOpen,
+            open: this.props.isOpen,
```

Comment:
You may also remove this property from the `state` entirely and make `ResizableSidebar` a controllable component.